### ISSUE TITLE
On Clicking Multiple times the "Copy link" button , we get multiple "Link copied" pop-up

### ISSUE THREAD

**Issue by aryamanpuri** [mozilla/voice-web#2574]
**Describe the bug**
On clicking the share link button , we get multiple pop-up of Link Copied

**To Reproduce**
1. Go to https://voice.mozilla.org
2. Record a clip
3. Click the upload button
![image](https://user-images.githubusercontent.com/43513114/75117381-e24cf980-5696-11ea-9c21-c5320cd9df4b.png)
4. Click the Copy link multiple times
You will come across
![Screenshot from 2020-02-23 23-27-25](https://user-images.githubusercontent.com/43513114/75117485-c1d16f00-5697-11ea-9648-27af28c7e38f.png)

** Expected behavior**
Pop up should come once irrespective of the number of times copy link is clicked

**Additional Behaviour**
Can I work on this bug ?

**Comment by MichaelKohler** [comment]
@aryamanpuri go for it :)

**Comment by aryamanpuri** [comment]
@MichaelKohler Thanks for replying .

I was thinking about the approach to be used in this bug .
What about your view on the use of simple if else condition.
For example 
 if(`addNotification`) then `nochange`  else `render notification`

Can you help me to get started to debug this ?

**Comment by MichaelKohler** [comment]
I don't have any knowledge about that part of the code, so I might not be the right person for this ;)

**Comment by aryamanpuri** [comment]
No Problem @MichaelKohler 
Can you help me with this bug @phirework

**Comment by rileyjshaw** [comment]
Hi @aryamanpuri, thanks for contributing to Common Voice. I just added some feedback to https://github.com/mozilla/voice-web/pull/2579. Our banner system was designed to show multiple notifications at once, to handle situations where several actions happen rapidly or concurrently. I think what you're seeing falls within expected behaviour.

I left some hints for how to tweak the rendering code in https://github.com/mozilla/voice-web/pull/2579, but first I would want more consensus that this is undesired behaviour.

@mbransn, do you have a quick reaction to which is the preferred behaviour?

- Showing all recent banners on screen at once
- Removing identical banners from displaying at the same time

**Comment by MichaelKohler** [comment]
> Removing identical banners from displaying at the same time

I'd definitely vote for that. Bonus points if there is some pulse or something so the user sees that it's a "refresh". But that's totally optional IMHO.

**Comment by aryamanpuri** [comment]
I agree with @MichaelKohler. Setting a timeout is not a bad option to limit the banners to one.
The changed code is working well in my local system.

**Comment by mbransn** [comment]
> > Removing identical banners from displaying at the same time
> 
> I'd definitely vote for that. Bonus points if there is some pulse or something so the user sees that it's a "refresh". But that's totally optional IMHO.

Yep, this @rileyjshaw would be my direction.

**Comment by aryamanpuri** [comment]
@phirework can you have a look at the pr and suggest is there any changes needed?
